tests/EdFi.Ods.Api.IntegrationTests/EducationOrganizationTestDataBuilder.cs

```diff
@@ -46,9 +48,56 @@ public static EducationOrganizationTestDataBuilder Initialize(IDbConnection conn
             command.CommandText = "SELECT GradeLevelDescriptorId FROM edfi.GradeLevelDescriptor;";
             builder.TestGradeLevelDescriptorId = Convert.ToInt32(command.ExecuteScalar());
 
+            command.CommandText = "SELECT EmploymentStatusDescriptorId FROM edfi.EmploymentStatusDescriptor;";
+            builder.TestEmploymentStatusDescriptorId = Convert.ToInt32(command.ExecuteScalar());
+
+            command.CommandText = "SELECT StaffClassificationDescriptorId FROM edfi.StaffClassificationDescriptor;";
+            builder.TestStaffClassificationDescriptorId = Convert.ToInt32(command.ExecuteScalar());
+
             return builder;
         }
 
+        public EducationOrganizationTestDataBuilder AddStaffEducationOrganizationEmploymentAssociation(int schoolId, int staffUSI, DateTime? entryDate = null)
+        {
+
+            if (!entryDate.HasValue)
+            {
+                entryDate = DateTime.UtcNow.Date;
+            }
+
+            _sql.AppendLine($@"INSERT INTO edfi.StaffEducationOrganizationEmploymentAssociation (EducationOrganizationId, StaffUSI, EmploymentStatusDescriptorId, HireDate)
```

Comment:
Could you please do formating similar to other places to code readability  example like public EducationOrganizationTestDataBuilder AddLocalEducationAgency(int localEducationAgencyId, int? stateEducationAgencyId = null, int? educationServiceCenterId = null, int? parentLocalEducationAgencyId = null) method